WordPress/Classes/ViewRelated/Site Creation/SiteIntent/SiteIntentData.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+struct SiteIntentData {
+    // Ordering of defaults affects ordering in the UI.
+    private static let defaultVerticalSlugs = [
```

Comment:
I don't have a strong opinion on this since the dataset is so small, so I've added the `isDefault` property. The `verticals` set is now an array, and I've moved the defaults up to the top and ordered them since they aren't alphabetical.